lib/inquirer.js

```diff
@@ -0,0 +1,14 @@
+const inquirer = require('inquirer');
+const questions = require('./observable-questions.js');
+
+inquirer.prompt(questions).ui.process.subscribe(
+	(ans) => {
+		console.log(ans);
```

Comment:
For yourself: Open in GH desktop: `npm link` could need a sudo flag. Type `webpack-cli --init`. Through jest: Simulate an inq instance if possible.